app/Http/Controllers/UserProjectsController.php

```diff
@@ -46,9 +45,11 @@ public function showContracted(): Response|View
         $user = Auth::user();
 
         if (in_array($user->context, ['individual', 'organization'])) {
-            if ($user->individual && $user->individual->isParticipant() && ($user->individual->isConsultant() || $user->individual->isConnector())) {
+            $userContext = $user->individual ?? $user->organization;
+            if ($userContext && ($userContext->isConsultant() || $userContext->isConnector() || $userContext->inProgressContractedProjects()->count())) {
```

Comment:
Looks like this is the second time this conditional is being checked. Makes me wonder if you should store these complex conditionals in variables such as `isParticipant` and `isContracted` or something like that.